fcrepo-kernel-impl/src/test/java/org/fcrepo/kernel/impl/TransactionImplTest.java

```diff
@@ -0,0 +1,173 @@
+/*
+ * Licensed to DuraSpace under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * DuraSpace licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.kernel.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import java.time.Instant;
+
+import org.fcrepo.persistence.api.PersistentStorageSession;
+import org.fcrepo.persistence.api.PersistentStorageSessionManager;
+import org.fcrepo.persistence.api.exceptions.PersistentStorageException;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+/**
+ * <p>
+ * TransactionTest class.
+ * </p>
+ *
+ * @author mohideen
+ */
+@RunWith(MockitoJUnitRunner.Silent.class)
+public class TransactionImplTest {
+
+    private TransactionImpl testTx;
+
+    @Mock
+    private TransactionManagerImpl txManager;
+
+    @Mock
+    private PersistentStorageSessionManager pssManager;
+
+    @Mock
+    private PersistentStorageSession psSession;
+
+    @Before
+    public void setUp() {
+        when(pssManager.getSession("123")).thenReturn(psSession);
+        when(txManager.getPersistentStorageSessionManager()).thenReturn(pssManager);
+        testTx = new TransactionImpl("123", txManager);
+    }
+
+    @Test
+    public void testGetId() {
+        assertEquals("123", testTx.getId());
+    }
+
+    @Test
+    public void testDefaultShortLived() {
+        assertEquals(true, testTx.isShortLived());
+    }
+
+    @Test
+    public void testSetShortLived() {
+        testTx.setShortLived(false);
+        assertEquals(false, testTx.isShortLived());
+    }
+
+    @Test
+    public void testCommit() {
+        testTx.commit();
+        try {
+            verify(psSession).commit();
+        } catch (PersistentStorageException e) {
+            e.printStackTrace();
```

Comment:
I'd suggest adding `throws Exception` to the test method instead to keep the test cleaner. The same holds true for all the other tests like this